features/cellular/framework/targets/UBLOX/AT/UBLOX_AT.cpp

```diff
@@ -22,8 +22,16 @@
 using namespace mbed;
 using namespace events;
 
+static const AT_CellularBase::SupportedFeature unsupported_features[] =  {
```

Comment:
I'm not too familiar with the cellular codebase, but why are these indicated as unsupported features?

And is there a reason they're now being enabled and/or supported?

Reply:
@ARMmbed/team-ublox

Reply:
In cellularInformation test case, `test_information_interface`, it is reading serial number with different parameters of `AT+CGSN` command and some Ublox modem does not support parameters with `AT+CGSN` command, resulting failure of test. So thats why added it as unsupported feature.